run/test/specs/disappearing_call.spec.ts

```diff
@@ -15,7 +15,7 @@ test.describe('Disappearing call test', () => {
 
 async function disappearingCallMessage1o1Ios(platform: SupportedPlatformsType) {
   const { device1, device2 } = await openAppTwoDevices(platform);
-  const time: DMTimeOption = DISAPPEARING_TIMES.ONE_MINUTE;
+  const time: DMTimeOption = DISAPPEARING_TIMES.THIRTY_SECONDS;
```

Comment:
```suggestion
  const time = DISAPPEARING_TIMES.THIRTY_SECONDS;
```

THIRTY_SECONDS should already be correctly typed